modules/upgrade-crd-helm.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * upgrading/upgrade-helm.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="upgrade-crd-helm{context}"]
+= Prepare CRD for upgrade
+
+[role="_abstract"]
+If upgrading from 4.6 or 4.7, the `SecurityPolicy` custom resource definition (CRD) needs to be prepared in order to avoid upgrade errors.
```

Comment:
```suggestion
If upgrading from version 4.6 or 4.7, you must prepare the `SecurityPolicy` custom resource definition (CRD) to avoid upgrade errors.
```